src/_P039_Thermosensors.ino

```diff
@@ -772,29 +802,18 @@ float readMax6675(struct EventStruct *event)
 float readMax31855(struct EventStruct *event)
 {
   P039_data_struct *P039_data = static_cast<P039_data_struct *>(getPluginTaskData(event->TaskIndex));
-  
-  union {
-      uint8_t messageBuffer[4] = {0x00, 0x00, 0x00, 0x00};
-      uint32_t value;
-  } u_mB;
-  
-  uint8_t CS_pin_no = get_SPI_CS_Pin(event);
 
-  // u_mB.messageBuffer[4] = {0x00, 0x00, 0x00, 0x00};
-
-  // "transfer" 0x0 and read the 32 Bit conversion register from the Chip
+  uint8_t messageBuffer[4] = {0x00, 0x00, 0x00, 0x00};
```

Comment:
See here the benefits of using `{0}` ;)

Reply:
True - i will add this to my personal programming toolbox, Thank You !

Reply:
Solved in upcoming push !